src/test/resources/test.properties

```diff
@@ -0,0 +1,8 @@
+url = https://trello.com/login
+boardName = my board
```

Comment:
There will be problems with storing test data in properties, if there is more than one boardName/listName etc used in tests. Please move it to another layer